eng/Version-Increment.ps1

```diff
@@ -0,0 +1,59 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER PackageCsprojPath
+Link to the package csproj
+
+.EXAMPLE
+Updating package version for core
+
+Version-Increment.ps1 -PackageCsprojPath "C:\Git\azure-sdk-for-net\sdk\core\Azure.Core\src\Azure.Core.csproj"
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Position=0)]
+    [ValidateNotNullOrEmpty()]
+    [string] $PackageCsprojPath
+)
+
+$VERSION_REGEX = "^(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?(\-(?<pre>[0-9A-Za-z\-\.]+))?(\+(?<build>[0-9A-Za-z\-\.]+))?$"
+$CsprojData = New-Object -TypeName XML
+$CsprojData.Load($PackageCsprojPath)
+
+$PackageVersion = Select-XML -Xml $CsprojData -XPath '/Project/PropertyGroup/Version'
+Write-Verbose "Current Version: ${PackageVersion}"
+
+$PackageVersion -Match $VERSION_REGEX 
```

Comment:
Similarly add a check to ensure we get a match. Just `if ($PackageVersion -match $VERSOION_REGEX)` and give and error if we don't match.